atd-vzd/migrations/migration_atd_txdot_crashes_2023_06_26--1622.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE atd_txdot_crashes
+ADD COLUMN in_austin_full_purpose boolean GENERATED ALWAYS AS (is_crash_in_austin_full_purpose(position, city_id)) STORED NOT NULL;
```

Comment:
when we close out this PR ill add this column and its meaning/how its calculated to our gitbook data dictionary